--- node_modules/vue/dist/vue.runtime.esm.js.bak	2019-03-16 00:32:50.688325600 +0900
+++ node_modules/vue/dist/vue.runtime.esm.js	2019-03-15 16:01:58.807192300 +0900
@@ -4245,16 +4245,15 @@
 // timestamp can either be hi-res (relative to page load) or low-res
 // (relative to UNIX epoch), so in order to compare time we have to use the
 // same timestamp type when saving the flush timestamp.
-if (
-  inBrowser &&
-  window.performance &&
-  typeof performance.now === 'function' &&
-  document.createEvent('Event').timeStamp <= performance.now()
-) {
-  // if the event timestamp is bigger than the hi-res timestamp
-  // (which is evaluated AFTER) it means the event is using a lo-res timestamp,
-  // and we need to use the lo-res version for event listeners as well.
-  getNow = function () { return performance.now(); };
+if (inBrowser) {
+  const performance = window.performance
+  if (
+    performance &&
+    typeof performance.now === 'function' &&
+    document.createEvent('Event').timeStamp <= performance.now()
+  ) {
+    getNow = () => performance.now()
+  }
 }
 
 /**
